### ISSUE TITLE
Wiki Please

### ISSUE THREAD

**Issue by pt-smd** [translationexchange/tml-js-express#1]
I am interested with this plugin, I found example below but it is js-core version:
https://github.com/translationexchange/tml-js/blob/master/samples/dom/public/index.html

is it the same approach? is it compatible with jade view?

**Comment by jackbit** [comment]
you can try this:

```
var TML =  require('tml-express');
var tmlExpress = TML.init(config.tr8n || 'enter_your_token_here');
app.use(tmlExpress);
```

**Comment by berk** [comment]
@domikado yes, it is compatible with EJS and Jade. I will post examples shortly.
